app/src/main/res/values/strings.xml

```diff
@@ -1,5 +1,25 @@
 <resources>
     <string name="app_name">caregiver</string>
-    <!-- TODO: Remove or change this placeholder text -->
     <string name="hello_blank_fragment">Hello blank fragment</string>
+
+    <string name="uuid_prompt">Enter UUID here</string>
+    <string name="region_prompt">Enter room name</string>
+    <string name="major_prompt">Major</string>
+    <string name="minor_prompt">Minor</string>
+    <string name="start_scanning">Start Scanning</string>
+    <string name="stop_scanning">Stop Scanning</string>
+    <string name="add_region">Add Room</string>
+
+    <!-- sign up / login -->
```

Comment:
I think if a string is only used in one component, can we stick it to the respective component only? We had merge conflict issues due to the fact that people are modifying this global file.